specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -300,6 +300,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "post": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "ListResourcePreparationErrors",
+        "consumes": [],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
```

Comment:
All APIs are required to have a "default" response (in addition to the 2XX success codes). `default` is used for all errors and should implement the standard error schema. [Here is an example.](https://github.com/Azure/azure-rest-api-specs/blob/2ea3a782af990bcc0918da995f9ab1c64490502c/specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json#L108)
Documentation for the error response contract: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content

Let's at least get this added in these new APIs in this PR

Reply:
Cool, fixed it.